### PR TITLE
Format FZF_DEFAULT_OPTS value

### PR DESCRIPTION
**Pull request by gerardbosch**
# Description

Properly formatting the options allow to print them nicely:
```
echo $FZF_DEFAULT_OPTS
```
will print one option per line, all with same indentation.

This makes understanding its value easier.

Here is the comparison before/after:

image

Signed-off-by: Gerard Bosch <gerard.bosch@gmail.com>

# License Acceptance

- [x] This repository is Apache version 2.0 licensed (some scripts may have alternate licensing inline in their code) and by making this PR, I am contributing my changes to the repository under the terms of the Apache 2 license.

### CODE REVIEW

File: fzf-zsh-plugin.plugin.zsh [hunk 1]
```diff
@@ -81,7 +81,7 @@ if [[ -z "$FZF_DEFAULT_COMMAND" ]]; then
   # If fd command is installed, use it instead of find
   _fzf_has 'fd' && _fd_cmd="fd"
   _fzf_has 'fdfind' && _fd_cmd="fdfind"
-  if [[ -n $_fd_cmd ]] then
+  if [[ -n "$_fd_cmd" ]]; then
```
Comment (gerardbosch) [inline fzf-zsh-plugin.plugin.zsh:84]: Wasn't that a syntax error? (the missing `;`) 🤔

Comment (unixorn) [reply fzf-zsh-plugin.plugin.zsh:84]: I think you're right, thanks for the fix

File: fzf-zsh-plugin.plugin.zsh [hunk 2]
```diff
@@ -100,29 +100,30 @@ fi
...
-  --preview-window=${FZF_PREVIEW_WINDOW}
-  --color='hl:148,hl+:154,pointer:032,marker:010,bg+:237,gutter:008'
-  --prompt='∼ ' --pointer='▶' --marker='✓'
-  --bind '?:toggle-preview'
-  --bind 'ctrl-a:select-all'
-  --bind 'ctrl-e:execute(vim {+} >/dev/tty)'
-  --bind 'ctrl-v:execute(code {+})'
-  "
-
+  fzf_default_opts+=(
```
Comment (unixorn) [inline fzf-zsh-plugin.plugin.zsh:103]: This is more idiomatic `zsh`, thanks

**Review by unixorn — APPROVED** [review]
Ready to go once the conflict in `README.md` is resolved.

File: README.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ Note - while these scripts could all be ZSH functions instead of scripts in the
 | `d-attach` | Uses `fzf` to select `docker` containers to start and attach to. | From the [fzf wiki](https://github.com/junegunn/fzf/wiki/examples) |
 | `d-image-rm` | Uses `fzf` to select `docker` containers to start and attach to. | From the [fzf wiki](https://github.com/junegunn/fzf/wiki/examples) |
 | `d-rm` | Uses `fzf` to select `docker` containers to remove. | From the [fzf wiki](https://github.com/junegunn/fzf/wiki/examples) |
-| `falias` | Searches your ZSH aliases, then puts your selection on the mac clipboard (macOS only) | |
```
Comment (unixorn) [inline README.md:42]: The `falias` entry needs to stay, but otherwise the readme lgtm

**Review by unixorn — APPROVED** [review]
lgtm

**Comment by unixorn** [comment]
Thanks for the contribution!

**Pull request merged**